pipelines.md

```diff
@@ -123,7 +123,7 @@ results/figures:
 ```
 
 ```{margin}
-`make` can be pretty finicky. For instance, it requires using tabs to indent and does not support spaces.
+`make` can be pretty finicky. For instance, you must use _tabs_, not spaces, to indent.
 ```
 
 The plot can be created with `make plot`. The `Makefile` contains a complete description of the inputs and outputs to different scripts, and thus serves as a self-documenting artifact. [Software carpentry](https://swcarpentry.github.io/make-novice/) has an excellent tutorial on `make`. What's more, `make` only rebuilds what needs to be rebuilt. In particular, if the network is already trained, `make` will detect it and won't retrain the network again, skipping ahead to the plotting task.
```

Comment:
How about adding an additional reference to https://makefiletutorial.com/ in this paragraph?

Reply:
I think I prefer the software carpentry link as is because it focuses on data science use cases rather than c++.